src/NumericalAlgorithms/Spectral/SwshCoefficients.hpp

```diff
@@ -8,9 +8,92 @@
 #include <sharp_cxx.h>
 
 #include "NumericalAlgorithms/Spectral/SwshCollocation.hpp"
+#include "Utilities/ForceInline.hpp"
 
 namespace Spectral {
 namespace Swsh {
+
+/// \ingroup SwshGroup
+/// \brief Convenience function for determining the number of spin-weighted
+/// spherical harmonics coefficients that are stored for a given `l_max`
+constexpr SPECTRE_ALWAYS_INLINE size_t
+number_of_swsh_coefficients(const size_t l_max) noexcept {
+  return (l_max + 1) * (l_max + 2) / 2;  // "triangular" representation
+}
+
+/*!
+ * \ingroup SwshGroup
+ * \brief Compute the relative sign change necessary to convert between the
+ * libsharp basis for spin weight `from_spin_weight` to the basis for spin
+ * weight `to_spin_weight`, for the real component coefficients if `real` is
+ * true, otherwise for the imaginary component coefficients.
+ *
+ * \details The sign change is obtained by the
+ * difference between the libsharp convention and the convention which uses:
+ *
+ * \f[
+ * {}_s Y_{\ell m} (\theta, \phi) = (-1)^m \sqrt{\frac{2 l + 1}{4 \pi}}
+ * D^{\ell}{}_{-m s}(\phi, \theta, 0).
+ * \f]
+ *
+ * See \cite Goldberg1966uu
+ */
+constexpr SPECTRE_ALWAYS_INLINE double sharp_swsh_sign_change(
+    const int from_spin_weight, const int to_spin_weight,
+    const bool real) noexcept {
+  if (real) {
+    return (from_spin_weight == 0 ? -1.0 : 1.0) *
+           (from_spin_weight >= 0 ? -1.0 : 1.0) *
+           ((from_spin_weight < 0 and (from_spin_weight % 2 == 0)) ? -1.0
+                                                                   : 1.0) *
+           (to_spin_weight == 0 ? -1.0 : 1.0) *
+           (to_spin_weight >= 0 ? -1.0 : 1.0) *
+           ((to_spin_weight < 0 and (to_spin_weight % 2 == 0)) ? -1.0 : 1.0);
+  }
+  return (from_spin_weight == 0 ? -1.0 : 1.0) *
+         ((from_spin_weight < 0 and (from_spin_weight % 2 == 0)) ? 1.0 : -1.0) *
+         (to_spin_weight == 0 ? -1.0 : 1.0) *
+         ((to_spin_weight < 0 and (to_spin_weight % 2 == 0)) ? 1.0 : -1.0);
+}
+
+/*!
+ * \ingroup SwshGroup
+ * \brief Compute the sign change between the libsharp convention and the set
```

Comment:
This should point to the docs at the beginning of TransformJob where the formula sYlm_sharp = sign_convention(s,m) sYlm_goldberg is given, and should say that it computes "sign_convention(s,m)" or whatever you want to call it.